mobility_pipeline/lib/make_matrix.py

```diff
@@ -36,6 +36,17 @@
 import numpy as np  # type: ignore
 import pandas as pd  # type: ignore
 
+from typing import List, Union, cast
+from mypy_extensions import TypedDict
+import numpy as np  # type: ignore
+from shapely.geometry import Polygon, MultiPolygon 
+from shapely.strtree import STRtree
```

Comment:
We use [`mypy`](http://mypy-lang.org/) to do type-checking, but it doesn't know how to handle some libraries, including `shapely`. We override its complaints by adding a `# type: ignore` after the import like this:
```suggestion
from shapely.strtree import STRtree  # type: ignore
```